Source/WebKit/UIProcess/ios/WebProcessProxyIOS.mm

```diff
@@ -71,6 +73,14 @@
     }
 
     throttler().setAllowsActivities(!m_processPool->processesShouldSuspend());
+
+#if ENABLE(REMOVE_XPC_AND_MACH_SANDBOX_EXTENSIONS_IN_WEBCONTENT)
+    if (WTF::CocoaApplication::isIBooks()) {
+        protectedProcessPool()->registerAssetFont("Canela Text"_s, *this);
+        protectedProcessPool()->registerAssetFont("Proxima Nova"_s, *this);
+        protectedProcessPool()->registerAssetFont("Publico Text"_s, *this);
+    }
```

Comment:
My understanding is only web process that loads page with preference `blockMobileAssetInWebContentSandbox()` true can get the font, but here we are giving it to all processes. Not sure if this is expected behavior.

Also with existing usage it seems  `registerAssetFont` does not need to take font -- it can decide the list by itself: like if isIBooks(), the list contains 3 items, else it's empty; similar to what we do for `registerUserInstalledFonts`.

Reply:
That is a good point! Since the preference `blockMobileAssetInWebContentSandbox()` is only a temporary preference meant for staging, and not a preference the client should be modifying, I think it is OK to set this for all processes. I expect the preference to be removed fairly soon. However, let me know if you think we should avoid this!